corehq/ex-submodules/casexml/apps/case/util.py

```diff
@@ -177,3 +177,35 @@ def iter_cases(case_ids, strip_history=False, wrap=True):
     else:
         for case in CommCareCase.bulk_get_lite(case_ids, wrap=wrap):
             yield case
+
+
+def get_date_case_property_changed(case, case_property_name, value):
+    """Returns the date a particular case property was changed to a specific value
```

Comment:
Might want to clarify that this gives the _first_ date a particular property was set to particular value.